common/features/org.jboss.tools.common.all.feature/feature.properties

```diff
@@ -1,9 +1,6 @@
 featureName=JBoss Tools Common All Feature
 featureProvider=JBoss by Red Hat
 
-# "updateSiteName" property - label for the update site
-updateSiteName=JBossTools Update Site
```

Comment:
How is this related to the license staff? Do we need to delete this in all features?